architectures

```diff
@@ -1,3 +1,4 @@
 bashbrew-arch   variants
 amd64     default,alpine,onbuild,slim,stretch,wheezy
 ppc64le default,onbuild,slim,stretch
+s390x default,alpine,onbuild,slim,stretch
```

Comment:
I removed `alpine`. However, I can build alpine variant in the s390x server. That's why I put alpine in there. Thanks for the information @tianon !